src/newProject.js

```diff
@@ -0,0 +1,21 @@
+var Sprite = require('./sprites/sprite');
+var Blocks = require('./engine/blocks');
+
+function newProject (runtime) {
```

Comment:
Rather than a separate file, it still feels preferable to me to have it as a method on VirtualMachine.prototype (index.js).